src/ProjectService.Data.Provider.MsSql.Ef/Migrations/20210819092000_UpdateDatabase.cs

```diff
@@ -103,7 +103,7 @@ private void AddTasksTable(MigrationBuilder migrationBuilder)
     private void AddTaskPropertiesTable(MigrationBuilder migrationBuilder)
     {
       migrationBuilder.CreateTable(
-        name: "TaskProperties",
+        name: "TasksProperties",
```

Comment:
why did you do this?

Reply:
Because in migrations there is confusion between TasksProperties and TaskProperties